sparse/__init__.py

```diff
@@ -116,6 +116,7 @@
     std,
     sum,
     tensordot,
+    var,
```

Comment:
This should also be added to `docs/generated/sparse.rst`.

Reply:
Done!